README.md

```diff
@@ -879,8 +879,7 @@ For running locally in container run
 
 1. `docker-compose -f docker-compose.yml build`
 2. `docker-compose -f docker-compose.yml up`
-3. `docker-compose -f docker-compose.metrics.yml build`
-4. `docker-compose -f docker-compose.metrics.yml up`
+3. `docker-compose -f docker-compose.metrics.yml up`
```

Comment:
README.MD
- Maybe move api outside to another .md file?
- -H 'accept: application/json' -> -H 'Accept: application/json'
- Just curl -X GET https://localhost:3005/v1/keys works, no need to specify additional headers
- Different response struc in /keys/find and in /modules
class Response {
  data: ModuleKey[];
  meta: {
    elBlockSnapshot: ElBlockSnapshot;
  };
}
and
class Reponse {
  data: Module[];
  elBlockSnapshot: ElBlockSnapshot;
}
- Maybe rewrite in classic style via json shcema or like in swagger.
- Is this endpoint useful `POST /v1/modules/{module_id}/keys/find` we have basic /find.
- Add db schema to readme
- Can't run tests with current guide. 38/79 tests are failed with
 EACCES: permission denied, scandir '/home/raman/work/lido-keys-api/.volumes/pgdata-1
or Timeouts